### DISCUSSION TITLE
Need file sorting capability in Web version

### DISCUSSION THREAD

**Discussion by szylllc** [community/community#32040] [Other Feature Feedback, Questions, & Ideas]
Currently, the file listing is only in alphabetic order. I need to sort by the date too.
See this for the background to this problem: https://github.com/orgs/community/discussions/31369

**Comment by Akash1134** [comment]
Thanks for sharing and we appreciate your patience. I have added the relevant label to the discussion so that the product team can review it. You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.
